cumulus/polkadot-parachain/src/command.rs

```diff
@@ -679,10 +765,12 @@ pub fn run() -> Result<()> {
 				cmd.run(config, polkadot_config)
 			})
 		},
-		Some(Subcommand::ExportGenesisState(cmd)) =>
-			construct_async_run!(|components, cli, cmd, config| {
-				Ok(async move { cmd.run(&*config.chain_spec, &*components.client) })
-			}),
+		Some(Subcommand::ExportGenesisState(cmd)) => {
+			let runner = cli.create_runner(cmd)?;
+			runner.sync_run(|config| {
+				construct_benchmark_partials!(config, |partials| cmd.run(&*config.chain_spec, &*partials.client))
```

Comment:
The macro should then be renamed.

Reply:
What about simply `construct_partials`?

If not, any suggestion?

On a side note: I've seen `construct_benchmark_partials` in several parachains and templates. If we are now to change the name (and I definitely agree with that, since I found it misleading in all cases I saw) we should take into account it will tend to be imitated elsewhere.

Reply:
> What about simply `construct_partials`?

Sounds good.

> we should take into account it will tend to be imitated elsewhere.

People copying our code/name is not really something that we should take into account here. This isn't any public interface.

Reply:
> Sounds good.

Done in commit [3a10545](https://github.com/paritytech/polkadot-sdk/pull/1521/commits/3a105458722afffbca152671ce00d8c96dd55b47).

> People copying our code/name is not really something that we should take into account here. This isn't any public interface.

Understood. Thanks for clarifying.